src/Hello.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+<div>
+    <h1>Hello Todays Weather </h1> 
+</div>
+
```

Comment:
Remember to remove this once you're done with it.  

You'd have to define "Hello" in here for the export to work.  At the moment you haven't defined anything called Hello.  

It looks like you want to make a react functional component.  If that's so you'd define it as a function.